web/src/locales/en-US/menu.ts

```diff
@@ -58,6 +58,7 @@ export default {
   'menu.account': 'Account',
   'menu.account.center': 'Account Center',
   'menu.account.settings': 'Account Settings',
+  'menu.account.setting.modify.settings': 'Settings',
```

Comment:
```suggestion
  'menu.account.modifySettings': 'Modify Settings',
```

Reply:
The little hump writing style doesn't seem to fit here, so I changed it to ‘menu.account.modify-settings': 'Modify Settings'

Reply:
Please keep using `Settings` here, for most dashboards, `Setting` is used universal.